sdk/typescript/package.json

```diff
@@ -0,0 +1,57 @@
+{
+    "name": "@fern-api/imdb",
+    "version": "0.0.9",
```

Comment:
Everytime you want to bump the version for your sdk you will have to manually bump this version (or you can setup some automation)